v2/object/string.go

```diff
@@ -1,29 +1,55 @@
 package object
 
-const (
-	typeRegularString      = "Regular"
-	typeTombstoneString    = "Tombstone"
-	typeStorageGroupString = "StorageGroup"
```

Comment:
Wonder if it break something :thinking:

Reply:
Relates to https://github.com/nspcc-dev/neofs-api-go/issues/310#issuecomment-864865838